app/ui/src/app/integration/list/list.component.scss

```diff
@@ -16,14 +16,19 @@
 
     .icon {
       position: relative;
-      .badge {
+
+      .syn-badge {
         position: absolute;
         background: #ec7a08;
```

Comment:
Any chance we can make this other color? I know that orange is the new black, but it screams like error to me, we can hope that it's me.

Reply:
@dongniwang what do you think?

Reply:
We’re going to gather user feedback on API provider and we can definitely get some feedback on the color of the badge. A more neutral color choice I can think of is perhaps blue? I’ll check with a visual design SME to get some suggestions.

Reply:
<img width="2017" alt="screen shot 2018-10-10 at 8 40 33 am" src="https://user-images.githubusercontent.com/35148959/46740347-3b65ea00-cc68-11e8-9cc2-6da90950fe0c.png">
<img width="2017" alt="screen shot 2018-10-10 at 8 40 26 am" src="https://user-images.githubusercontent.com/35148959/46740348-3b65ea00-cc68-11e8-95ac-3ed3c771f9f3.png">
<img width="2017" alt="screen shot 2018-10-10 at 8 39 46 am" src="https://user-images.githubusercontent.com/35148959/46740349-3bfe8080-cc68-11e8-8898-c33e06f3d52d.png">

@dongniwang that sounds good to me. here's a screenshot of the dark and light blue, and one with the list row hover state (blue background)